src/test/resources/META-INF/mods.toml

```diff
@@ -42,4 +42,8 @@ loaderVersion="[28,)"
 [[mods]]
     modId="command_event_test"
 [[mods]]
-    modId="entity_selector_test"
\ No newline at end of file
+    modId="entity_selector_test"
+[[mods]]
+    modId="pistoneventtest"
```

Comment:
The pistoneventtest is included in the test package so without this the game does not load